templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -132,7 +132,14 @@ class IKFastKinematicsPlugin : public kinematics::KinematicsBase
   /** @class
    *  @brief Interface for an IKFast kinematics plugin
    */
-  IKFastKinematicsPlugin():active_(false){}
+  IKFastKinematicsPlugin():
+    active_(false)
+  {
+    srand( time(NULL) );
```

Comment:
Is this the responsibility of the plugin, or of the parent process? It is highly likely srand() is already being called in the parent process, as is the case with MoveIt! (which this plugin is only made for)